pkg/operator/ceohelpers/already_synced_informer.go

```diff
@@ -0,0 +1,21 @@
+package ceohelpers
+
+import (
+	"k8s.io/client-go/tools/cache"
+)
+
+// AlreadySyncedInformer wraps an ordinary cache.SharedInformer, but allows us to fake HasSynced results.
+// This is to allow controllers to proceed when an informer could not sync (eg when the API isn't available).
+type AlreadySyncedInformer struct {
```

Comment:
leaving this in here for now, might need to reuse this for the optional machine API